### PR TITLE
Arg data

### PR DESCRIPTION
**Pull request by ChouaibB**
Added data files, script for making ARG data and the TreeSE object.

### CODE REVIEW

File: inst/extdata/arg_data/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+And these URL sources:
+ 
+1) Sample metadata
+https://github.com/kihyunee/gut_resistotype/blob/data/sfile-microbiome-sample-detail.tsv
+
+2) read-based normalized ARG abundance table
+https://zenodo.org/records/7383076/files/cpg_table.20220516_scg_valid_only.tsv
+
+3) assembly-based normalized ARG abundance table
+https://zenodo.org/records/7383076/files/DS3.SCG_normalized_ARG_abund.columns_CARD_ref.n_6104.tsv
```
Comment (antagomir) [inline inst/extdata/arg_data/README.md:15]: There were two more files - could them be linked here, too?

File: inst/extdata/arg_data/main.R [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+
+library(ggplot2)
+p <- ggplot(d, aes(x=read, y=assembly)) + geom_point(size=1, alpha=0.5) +
+       scale_x_log10() +
+       scale_y_log10()        
+print(p)
+
+
+
```
Comment (antagomir) [inline inst/extdata/arg_data/main.R:16]: The main.R file can be removed

**Review by antagomir — COMMENTED** [review]
Very helpful, thanks! Just the minor comments

File: inst/extdata/arg_data/README.md (hunk 1 above)
Comment (ChouaibB) [reply inst/extdata/arg_data/README.md:15]: One I have downloaded from what seems a personal drive, the second also was downloaded without a source or link. Do you have the links?

Comment (antagomir) [reply inst/extdata/arg_data/README.md:15]: Ah, ok. List the file names here, and write "Received from Kihyun Lee on April 19, 2024 (personal communication)."

**Review by antagomir — APPROVED** [review]

**Comment by antagomir** [comment]
@ChouaibB - sorry for the hassle .. but I noticed that we already had a data repository here:
https://github.com/microbiome/data

-> Could you update the PR to include this as a new folder in that repository..? Name of the subfolder could be changed to Lee2023.

**Pull request closed without merging**